tsdb/chunks/head_chunks.go

```diff
@@ -273,6 +273,26 @@ func NewChunkDiskMapper(reg prometheus.Registerer, dir string, pool chunkenc.Poo
 	return m, m.openMMapFiles()
 }
 
+// Chunk encodings for out-of-order chunks.
+// These encodings must be only used by the Head block for its internal bookkeeping.
+const (
+	OutOfOrderMask = 0b10000000
```

Comment:
This should be uint8 to begin with and don't need typecasting elsewhere, else it can lead buggy code if not used correctly.
```suggestion
	OutOfOrderMask = uint8(0b10000000)
```

Reply:
Done